storybookjs/storybook
# Vue3: Add default render function CSF3

Issue: #15498

Greetings.

## What I did

In this pull-request I'm created a default `render()` function for Vue 3 and provided an example of usage with CSF3 story.

## How to test


### Testing

1. Build  `@storybook/vue3` package:

    ```
    yarn build vue3
    ```

2. Run Vue 3 CLI example:

    ```
    cd examples/vue-3-cli-example
    yarn storybook
    ```

3. Open `ButtonCSF3` Storybook page in a browser.
You will see a rendered Vue 3 component.

Best wishes,
Sergey.
